config/config.exs

```diff
@@ -189,6 +189,15 @@ config :oli, :footer,
   link_2_location: System.get_env("FOOTER_LINK_2_LOCATION", ""),
   link_2_text: System.get_env("FOOTER_LINK_2_TEXT", "")
 
+# Configure libcluster for horizontal scaling
+# Take into account that different strategies could use different config options
+config :libcluster,
+  topologies: [
+    oli: [
+      strategy: Module.concat([System.get_env("LIBCLUSTER_STRATEGY", "Cluster.Strategy.Gossip")])
```

Comment:
Is it necessary to use `Module.concat` here? We are passing a one-element-list and I think it's used when you want to concat several aliases.

Reply:
I totally agree but I didn't find a quick way to build a module name from a string, I hear suggestions 😄